pkg/pod/pod_test.go

```diff
@@ -108,6 +108,8 @@ func TestPodBuild(t *testing.T) {
 					"/tekton/tools/0",
 					"-termination_path",
 					"/tekton/termination",
+					"-timeout",
```

Comment:
Looking through these test cases it doesn't seem like there are any that check that a step that specifies `timeout` results in a pod container that specifies the `-timeout` flag with the same value -- all the tests have `"-timeout", ""`. Could you add one, to guard against future breakages?